apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/SummaryPage.tsx

```diff
@@ -105,8 +108,8 @@ export const SummaryPage = ({
   const searchParams = useSearchParams();
 
   survey = useMemo(() => {
```

Comment:
Please don't mutate the function parameter `survey`, you can make a new variable here `surveyMemoized` and use it in the rest of the places 😊 

```
const surveyMemoized = useMemo(() => {
    return checkForRecallInHeadline(survey, "default", attributeClasses);
  }, [survey, attributeClasses]);
```